src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -289,14 +287,19 @@ const SelfExclusion = (() => {
     const additionalCheck = data => (
         new Promise((resolve) => {
             const is_changed = Object.keys(data).some(key => ( // using != in next line since response types is inconsistent
-                key !== 'set_self_exclusion' && (!(key in self_exclusion_data) || self_exclusion_data[key] != data[key]) // eslint-disable-line eqeqeq
+                key !== 'set_self_exclusion' && (self_exclusion_data[key] != data[key] && data[key] !== '') || (self_exclusion_data[key] !== undefined && data[key] === '')// eslint-disable-line eqeqeq
```

Comment:
why not do typeof undefined so we don't hit the eslint error?

Reply:
I changed as suggested, however, it wasn't the reason causing eslint error. This happens because of `!=` operator as mentioned.